tests/integration/dbapi/async/V2/test_timeout.py

```diff
@@ -0,0 +1,12 @@
+from pytest import raises
+
+from firebolt.async_db import Connection
+from firebolt.utils.exception import QueryTimeoutError
+
+from .test_queries_async import LONG_SELECT
```

Comment:
Let's use either relative or absolute imports like in this tests's [sync](https://github.com/firebolt-db/firebolt-python-sdk/pull/410/files#diff-5331bb2e13e9798e8f79a7f028be771940804514f551849da90c7313c1a7cbb6R1) counterpart.

Reply:
The problem here is that having a word `async` in import is treated as keyword and not as a module name. That's why I made id relative here.

Reply:
Made it relative in sync as well